### ISSUE TITLE
Warnings with PHP8 / Dokuwiki 2023-04-04a "Jack Jackrum"

### ISSUE THREAD

**Issue by Miro-Collas** [dokufreaks/plugin-tag#252]
```
2023-09-07 22:57:54E_WARNING: Undefined array key "name"/home/fswiki/public_html/lib/plugins/tag/helper.php(556)
    #0 /home/fswiki/public_html/lib/plugins/tag/helper.php(556): dokuwiki\ErrorHandler::errorHandler(2, 'Undefined array...', '/home/fswiki/pu...', 556)
    #1 /home/fswiki/public_html/lib/plugins/tag/helper.php(535): helper_plugin_tag->replacePlaceholders('ms_security')
    #2 [internal function]: helper_plugin_tag->cleanTag('ms_security')
    #3 /home/fswiki/public_html/lib/plugins/tag/helper.php(524): array_map(Array, Array)
    #4 /home/fswiki/public_html/lib/plugins/tag/helper.php(374): helper_plugin_tag->cleanTagList(Array)
    #5 /home/fswiki/public_html/lib/plugins/cloud/syntax.php(276): helper_plugin_tag->tagOccurrences(Array, Array, true, 1)
    #6 /home/fswiki/public_html/lib/plugins/cloud/syntax.php(80): syntax_plugin_cloud->_getTagCloud('600', NULL, NULL, Array, Object(helper_plugin_tag))
    #7 /home/fswiki/public_html/inc/parser/renderer.php(119): syntax_plugin_cloud->render('xhtml', Object(Doku_Renderer_xhtml), Array)
    #8 /home/fswiki/public_html/inc/parserutils.php(700): Doku_Renderer->plugin('cloud', Array, 5, '~~TAGCLOUD:600~...')
    #9 /home/fswiki/public_html/inc/parserutils.php(158): p_render('xhtml', Array, NULL)
    #10 /home/fswiki/public_html/inc/parserutils.php(96): p_cached_output('/home/fswiki/pu...', 'xhtml', 'fs_cloud')
    #11 /home/fswiki/public_html/inc/Ui/PageView.php(68): p_wiki_xhtml('fs_cloud', 0, true, '')
    #12 /home/fswiki/public_html/inc/Action/Show.php(37): dokuwiki\Ui\PageView->show()
    #13 /home/fswiki/public_html/inc/template.php(100): dokuwiki\Action\Show->tplContent()
    #14 [internal function]: tpl_content_core('show')
    #15 /home/fswiki/public_html/inc/Extension/Event.php(133): call_user_func_array('tpl_content_cor...', Array)
    #16 /home/fswiki/public_html/inc/Extension/Event.php(199): dokuwiki\Extension\Event->trigger('tpl_content_cor...', true)
    #17 /home/fswiki/public_html/inc/template.php(85): dokuwiki\Extension\Event::createAndTrigger('TPL_ACT_RENDER', 'show', 'tpl_content_cor...')
    #18 /home/fswiki/public_html/lib/tpl/bootstrap3/main.php(117): tpl_content(false)
    #19 /home/fswiki/public_html/inc/actions.php(27): include('/home/fswiki/pu...')
    #20 /home/fswiki/public_html/doku.php(126): act_dispatch()
    #21 {main}
2023-09-07 22:57:54E_WARNING: Uninitialized string offset 0/home/fswiki/public_html/lib/plugins/tag/helper.php(558)
    #0 /home/fswiki/public_html/lib/plugins/tag/helper.php(558): dokuwiki\ErrorHandler::errorHandler(2, 'Uninitialized s...', '/home/fswiki/pu...', 558)
    #1 /home/fswiki/public_html/lib/plugins/tag/helper.php(535): helper_plugin_tag->replacePlaceholders('ms_security')
    #2 [internal function]: helper_plugin_tag->cleanTag('ms_security')
    #3 /home/fswiki/public_html/lib/plugins/tag/helper.php(524): array_map(Array, Array)
    #4 /home/fswiki/public_html/lib/plugins/tag/helper.php(374): helper_plugin_tag->cleanTagList(Array)
    #5 /home/fswiki/public_html/lib/plugins/cloud/syntax.php(276): helper_plugin_tag->tagOccurrences(Array, Array, true, 1)
    #6 /home/fswiki/public_html/lib/plugins/cloud/syntax.php(80): syntax_plugin_cloud->_getTagCloud('600', NULL, NULL, Array, Object(helper_plugin_tag))
    #7 /home/fswiki/public_html/inc/parser/renderer.php(119): syntax_plugin_cloud->render('xhtml', Object(Doku_Renderer_xhtml), Array)
    #8 /home/fswiki/public_html/inc/parserutils.php(700): Doku_Renderer->plugin('cloud', Array, 5, '~~TAGCLOUD:600~...')
    #9 /home/fswiki/public_html/inc/parserutils.php(158): p_render('xhtml', Array, NULL)
    #10 /home/fswiki/public_html/inc/parserutils.php(96): p_cached_output('/home/fswiki/pu...', 'xhtml', 'fs_cloud')
    #11 /home/fswiki/public_html/inc/Ui/PageView.php(68): p_wiki_xhtml('fs_cloud', 0, true, '')
    #12 /home/fswiki/public_html/inc/Action/Show.php(37): dokuwiki\Ui\PageView->show()
    #13 /home/fswiki/public_html/inc/template.php(100): dokuwiki\Action\Show->tplContent()
    #14 [internal function]: tpl_content_core('show')
    #15 /home/fswiki/public_html/inc/Extension/Event.php(133): call_user_func_array('tpl_content_cor...', Array)
    #16 /home/fswiki/public_html/inc/Extension/Event.php(199): dokuwiki\Extension\Event->trigger('tpl_content_cor...', true)
    #17 /home/fswiki/public_html/inc/template.php(85): dokuwiki\Extension\Event::createAndTrigger('TPL_ACT_RENDER', 'show', 'tpl_content_cor...')
    #18 /home/fswiki/public_html/lib/tpl/bootstrap3/main.php(117): tpl_content(false)
    #19 /home/fswiki/public_html/inc/actions.php(27): include('/home/fswiki/pu...')
    #20 /home/fswiki/public_html/doku.php(126): act_dispatch()
    #21 {main}
```

**Comment by Klap-in** [comment]
Fixed in #253 and #254, thanks!

**Comment by Miro-Collas** [comment]
@Klap-in  Thanks very much!
